src/targets/maven.ts

```diff
@@ -55,14 +59,21 @@ const targetOptions = [
 ] as const;
 type OptionsType = typeof targetOptions[number];
 
+type AndroidFields = {
+  android: {
```

Comment:
Isn't this an optional field?
```suggestion
  android?: {
```

Reply:
No, the android configuration is required. It's the only way to get the release files (without traversing through the directories).